### DISCUSSION TITLE
AtlasMap stopped working after one of the latest Chromium updates in browsers based on its engine

### DISCUSSION THREAD

**Discussion by lyakot** [atlasmap/atlasmap#4624] [Q&A]
Any thoughts on how to fix this problem?

![image](https://github.com/user-attachments/assets/a18dcb13-3bf5-45c2-b0f6-612013254455)

**Comment by arockiaraj1994** [comment]
I am also facing same issue. For workaround use firefox.

**Comment by mattrpav** [comment]
Please open an issue. The project needs a new UI-layer lead developer to step in to help with maintenance.

I suspect this is an update-to-latest JS type issue.

**Reply by arockiaraj1994** [reply]
Agreed. We need to really get into and maintain the project

**Comment by lyakot** [comment]
The problem was solved by updating ky to version 0.33.3
